instat/dlgOneWayFrequencies.vb

```diff
@@ -21,8 +21,7 @@ Public Class dlgOneWayFrequencies
     Private bReset As Boolean = True
     Private bResetSubdialog As Boolean = False
     Private clsSjTab As New RFunction
-    Private clsSjPlot, clsPlotGrid, clsSelect As New RFunction
-    Private clsTableBaseOperator, clsGraphBaseOperator As New ROperator
+    Private clsSjPlot, clsPlotGrid As New RFunction

```

Comment:
Yes, that will be quite helpful so that we agree on the  approach to use for all the dialogues. Adding Roger to the discussion might be helpful I suppose.